makefiles/boards/alias.inc.mk

```diff
@@ -0,0 +1,9 @@
+ALIAS = linux phynode
+
+# this basically implements a look-up table, is there a nicer way in make?
+ifneq (,$(filter phynode,$(BOARD)))
+  BOARD := pba-d-01-kw2x
```

Comment:
This needs `override` in order to be able to replace the BOARD setting if given on the command line: `make BOARD=linux`, instead of `BOARD=linux make` or `export BOARD=linux`

Reply:
it worked without the override, but maybe that war more a coincidence then? Anyway, makes much sense and I will fix this.